Source/JavaScriptCore/tools/JSDollarVM.cpp

```diff
@@ -743,15 +743,11 @@ JSC_DEFINE_CUSTOM_SETTER(testStaticAccessorPutter, (JSGlobalObject* globalObject
     return thisObject->putDirect(vm, PropertyName(Identifier::fromString(vm, "testField"_s)), JSValue::decode(value));
 }
 
-static const struct CompactHashIndex staticCustomAccessorTableIndex[9] = {
-    { -1, -1 },
+static const struct CompactHashIndex staticCustomAccessorTableIndex[5] = {
+    { 0, 4 },
```

Comment:
Are they correctly updated with 72 threshold?

Reply:
Yes, just re-computed and it's correct.